drivers/postgres.go

```diff
@@ -67,16 +67,8 @@ func (db *Postgres) GetDatabases() (databases []string, err error) {
 	if err != nil {
 		return nil, err
 	}
-
 	defer rows.Close()
```

Comment:
You mean `(*sql.Rows).Err()`? It's not useful here.

If the query fails, the error is returned by `(*sql.DB).Query`.
If the cursor inside `(*sql.Rows)` gets broken, the `(*sql.Rows).Next()` returns `false` and the error is checked by `(*sql.Rows).Err()` later.

Reply:
No I'm talking about the fact the err returned by Close is not checked because of the defer.

The code doesn't log things, so such error should be caught I think.

Reply:
But we got the data. I think we can shrug on this error. Otherwise, we would drop the results just because we lost connection in the very last moment?
It's like forgetting everything a friend said to you on the phone, but instead of hanging up, the connection eventually disconnects.